docs/tre-admins/setup-instructions/configuring-shared-services.md

```diff
@@ -1,22 +1,112 @@
+
+This will invoke the certs service to use Letsencrypt to generate a certificate for the specified domain prefix followed by `-{TRE_ID}.{LOCATION}.cloudapp.azure.com`, so in our case, having entered `nexus`, this will be `nexus-{TRE_ID}.{LOCATION}.cloudapp.azure.com`, which will be the public domain for our Nexus service.
+
+Once this has completed, you can verify its success either from the operation output, or by navigating to your core keyvault (`kv-{TRE_ID}`) and looking for a certificate called `nexus-ssl` (or whatever you called it).
+
+After verifying the certificate has been generated, you can deploy Nexus:
+
+1. Run the below commands in your terminal to build, publish and register the Nexus shared service bundle:
+
+  ```cmd
+  make bundle-build DIR=./templates/shared_services/sonatype-nexus-vm
+  make bundle-publish DIR=./templates/shared_services/sonatype-nexus-vm
+  make bundle-register DIR=./templates/shared_services/sonatype-nexus-vm BUNDLE_TYPE=shared_service
+  ```
+
+1. Navigate to the Swagger UI for your TRE API at `https://<azure_tre_fqdn>/api/docs`, and authenticate if you haven't already by clicking `Authorize`.
+
+1. Click `Try it out` on the `POST` `/api/shared-services` operation, and paste the following to deploy the Nexus shared service:
+
+  ```json
+  {
+    "templateName": "tre-shared-service-sonatype-nexus",
+    "properties": {
+      "display_name": "Nexus",
+      "description": "Proxy public repositories with Nexus",
+      "ssl_cert_name": "nexus-ssl"
+    }
+  }
+  ```
+
+!!! tip
+    If you called your cert something different in the certs shared service step, make sure that is reflected above.
+
+This will deploy the infrastructure required for Nexus, then start the service and configure it with the repository configurations located in the `./templates/shared_services/sonatype-nexus-vm/scripts/nexus_repos_config` folder. It will also set up HTTPS using the certificate you generated in the previous section, so proxies can be served at `https://nexus-{TRE_ID}.{LOCATION}.cloudapp.azure.com`.
+
+You can optionally go to the Nexus web interface by visiting `https://nexus-{TRE_ID}.{LOCATION}.cloudapp.azure.com/` in the jumpbox and signing in with the username `admin` and the password secret located in your core keyvault, with the key `nexus-admin-password`. Here you should be able to see all of the configured repositories and you can use the UI to manage settings etc.
+
+Just bear in mind that if this service is redeployed any changes in the UI won't be persisted. If you wish to add new repositories or alter existing ones, use the JSON files within the `./nexus_repos_config` directory.
+
+### B. Migrate from existing Nexus service (hosted on App Service)
```

Comment:
Don't think this will work as the workspace will be missing the DNS zone.

Reply:
Have added a note requiring workspace version `0.2.14` or above